### PR TITLE
Update hugepage related test cases to avoid extra_params

### PR DESCRIPTION
**Pull request by huangyum**
Signed-off-by: Yumei Huang yuhuang@redhat.com
ID: 1926689
Depends on https://github.com/avocado-framework/avocado-vt/pull/3073.

### CODE REVIEW

**Comment by huangyum** [comment]
Ran huge_page test loop with both qemu6.0 and qemu4.2, all cases passed with correct qemu cli.
@yfu Pls help review, thanks.

File: qemu/tests/slof_hugepage.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def _setup_hugepage(params):
     error_context.context('mount hugepages to %s'
                           % huge_page.hugepage_path, logging.info)
     huge_page.mount_hugepage_fs()
+    params["hugepage_path"] = huge_page.hugepage_path
```
Comment (yanan-fu) [inline qemu/tests/slof_hugepage.py:43]: Did you test this one, for ppc platform ?

Comment (huangyum) [reply qemu/tests/slof_hugepage.py:43]: Yes, passed with qemu4.2 and 6.0.

**Comment by yanan-fu** [comment]
LGTM

Waiting for avocado-vt patch.

**Comment by yanan-fu** [comment]
avocado-vt have merged, so merge this one!

Ack

**Pull request merged**